source/reference/minio-server/minio-server.rst

```diff
@@ -2198,6 +2198,39 @@ endpoints as ``PRIMARY`` and ``SECONDARY`` respectively:
    :mc-conf:`notify_webhook comment <notify_webhook.comment>`
    configuration setting.
 
+.. _minio-server-envvar-object-lambda-webhook:
+
+Object Lambda
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following section documents environment variables for configuring MinIO to
+publish data to an HTTP webhook endpoint for an Object Lambda function. See
+:ref:`developers-object-lambda` for more complete documentation and
+tutorials on using these environment variables.
+
+You can specify multiple webhook endpoints as Lambda targets by appending
+a unique identifier ``_FUNCTIONNAME`` for each Object Lambda function.
+For example, the following command set two distinct Object Lambda webhook endpoints:
+
+.. code-block:: shell
+   :class: copyable
+
+   export MINIO_LAMBDA_WEBHOOK_ENABLE_myfunction="on"
+   export MINIO_LAMBDA_WEBHOOK_ENDPOINT_myfunction="http://webhook-1.example.net"
+   export MINIO_LAMBDA_WEBHOOK_ENABLE_yourfunction="on"
+   export MINIO_LAMBDA_WEBHOOK_ENDPOINT_yourfunction="http://webhook-2.example.net"
+
+.. envvar:: MINIO_LAMBDA_WEBHOOK_ENABLE
+
+   Specify ``"on"`` to enable publishing :mc:`minio server` logs to the Object Lambda
+   webhook endpoint for the specified function.
+
+   Requires specifying :envvar:`MINIO_LAMBDA_WEBHOOK_ENDPOINT`.
+
+.. envvar:: MINIO_LAMBDA_WEBHOOK_ENDPOINT
+
+   The HTTP endpoint of the webhook for the specified function.
```

Comment:
If `MINIO_LAMBDA_WEBHOOK_ENABLE` requires `MINIO_LAMBDA_WEBHOOK_ENDPOINT`, is the reverse true? (Or maybe it's fine to define but not enable.)

Reply:
In other places, you can define something and then enable/disable it separately. But you can only enable it if it is defined.